datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -90,9 +90,9 @@ DROP TABLE IF EXISTS dcsa_im_v3_0.transport_call CASCADE;
 CREATE TABLE dcsa_im_v3_0.transport_call (
     id varchar(100) PRIMARY KEY,
     transport_call_sequence_number integer,
-    facility_code varchar(11) NULL REFERENCES dcsa_im_v3_0.facility (facility_code),
+    facility uuid NULL REFERENCES dcsa_im_v3_0.facility (id),
     facility_type_code char(4) NULL REFERENCES dcsa_im_v3_0.facility_type (facility_type_code),
-    other_facility varchar(50) NULL,
+    other_facility uuid NULL REFERENCES dcsa_im_v3_0.facility (id),
```

Comment:
Is it your intention that the location objects fits the need of varchar(50)? Because we don't have a freetext field in neither facility or location